execution-service/src/main/java/org/ihtsdo/rvf/execution/service/config/ValidationRunConfig.java

```diff
@@ -134,6 +136,15 @@ public ValidationRunConfig setForm(CharacteristicType form) {
 		return this;
 	}
 
+	public final String getResponseQueue() {
+		return responseQueue;
+	}
+
+	public final ValidationRunConfig setResponseQueue(final String responseQueue) {
+		this.responseQueue = responseQueue;
```

Comment:
I would rename this method name to addResponseQueue

Reply:
Fixed.